scene/gui/tree.h

```diff
@@ -587,6 +589,7 @@ class Tree : public Control {
 		int children_hl_line_width = 0;
 		int parent_hl_line_margin = 0;
 		int draw_guides = 0;
+		int draw_stripes = 0;
```

Comment:
This should be a `bool`.

Reply:
The theme cache constants can be bools, even if they are treated like integers by the theme system.
Example of bool in ThemeCache of another class:
https://github.com/godotengine/godot/blob/7c6e85de0dd7627d1a43acf5f144cee34df6ab36/scene/gui/color_picker.h#L227
I think the ones in Tree were missed during the refactor.

Reply:
Ah, yeah I was following draw_guides but can change it